### ISSUE TITLE
Add new regions/countries to countries list

### ISSUE THREAD

**Issue by dew54** [mapsme/omim#12169]
Hello everyone, i was wondering..
How could i insert new regions in the list of possible countries?

I know that there is the countries.txt file, where are specified the .poly file for regions, but i think that is not enough.. 

Please could you give me an help?

**Comment by mpimenov** [comment]
> but i think that is not enough

What makes you think so?

Add a region to data/borders/: https://github.com/mapsme/omim/tree/74f1d5b41f5395346a51072f300f45ae9bc49703/data/borders

Then regenerate packed_polygons.bin and countries.txt:
https://github.com/mapsme/omim/blob/74f1d5b41f5395346a51072f300f45ae9bc49703/data/packed_polygons.bin
https://github.com/mapsme/omim/blob/74f1d5b41f5395346a51072f300f45ae9bc49703/data/countries.txt

**Comment by dew54** [comment]
Hi @mpimenov , thanks for your support!

So, i'm trying this:

`cd  ~/omim-build-release`
`./generator_tool --generate_packed_borders`

And i get this:

`terminate called after throwing an instance of 'FileAbsentException'
  what():  FileAbsentException /home/dev/omim/platform/platform.cpp:149, "File types.txt doesn't exist in the scope wrf Have been looking in:
 /
/
/home/dev/.config/MapsWithMe/"
Core exception: File types.txt doesn't exist in the scope wrf Have been looking in:
 /
/
/home/dev/.config/MapsWithMe/
 0# generator::ErrorHandler(int) in ./generator_tool
 1# 0x00007FB49D96E440 in /lib/x86_64-linux-gnu/libc.so.6
 2# gsignal in /lib/x86_64-linux-gnu/libc.so.6
 3# abort in /lib/x86_64-linux-gnu/libc.so.6
 4# 0x00007FB49E344957 in /usr/lib/x86_64-linux-gnu/libstdc++.so.6
 5# 0x00007FB49E34AAB6 in /usr/lib/x86_64-linux-gnu/libstdc++.so.6
 6# 0x00007FB49E34AAF1 in /usr/lib/x86_64-linux-gnu/libstdc++.so.6
 7# 0x00007FB49E34AD24 in /usr/lib/x86_64-linux-gnu/libstdc++.so.6
 8# Platform::ReadPathForFile(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >) const in ./generator_tool
 9# Platform::GetReader(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) const in ./generator_tool
10# classificator::Load() in ./generator_tool
11# 0x0000559799559AE0 in ./generator_tool
12# main in ./generator_tool
13# __libc_start_main in /lib/x86_64-linux-gnu/libc.so.6
14# _start in ./generator_tool
Aborted
`

This is weird, becouse on MacOs no errors are throwed, but it's an macOs issue probably,
So, how do i generate packed _polygons.bin?

Thank you very much, 
Davide

**Comment by mpimenov** [comment]
This error message tells you exactly what's wrong: the required file is not found in any of the directories that the program tried. Search the project code for ```FileAbsentException``` to understand how these directories are chosen.

**Comment by dew54** [comment]
Ok, so, i've succesfully created the packed_polygon.bin file, thanks a lot! :)

Now, i should generate the countries.txt file, i thougt to do it manually, but i see it's not so easy.. So i guess there is another tool to create it, right?

**Comment by mpimenov** [comment]
Yes: https://github.com/mapsme/omim/issues/12147#issuecomment-559425883

**Comment by dew54** [comment]
Well, i'm trying to create countries.txt file, i am doing:
`cd ./omim/tools/python/post_generation`
and tried:
`python ./hierarchy_to_countries.py`
wich returns:
`  File "hierarchy_to_countries.py", line 42
    filename = os.path.join(path, f"{name}.mwm")
                                              ^
SyntaxError: invalid syntax`

And also tried:
`python3.7 ./hierarchy_to_countries.py`

wich return anything.

I'm missing something? Thanks a lot again.
Davide :)
